### ISSUE TITLE
Meeting Times

### ISSUE THREAD

**Issue by indolering** [namecoin/namecoin.info#88]
We need to publish meeting times on the website.  As meetings are on the first and third Saturday of every month at 5PM UTC, it requires two different Google calendar events:

* [First Saturday](https://www.google.com/calendar/event?action=TEMPLATE&tmeid=cmZldmY1N3V1dDJsZzkzMDZiaHVsdnIwYzRfMjAxNTAyMDdUMTcwMDAwWiBpbmRvbGVyaW5nQG0&tmsrc=indolering%40gmail.com).
* [Third Saturday](https://www.google.com/calendar/event?action=TEMPLATE&tmeid=bjBzbmVyN29wbjAyNnFvcGtzdGtkOTZwcmdfMjAxNTAxMTdUMTcwMDAwWiBpbmRvbGVyaW5nQG0&tmsrc=indolering%40gmail.com)

**Comment by JeremyRand** [comment]
You're forgetting that a large part of our userbase doesn't want Google tracking them.  It should be possible to export an iCal file with the relevant events and host it entirely on our website.

**Comment by indolering** [comment]
> You're forgetting that a large part of our userbase doesn't want Google tracking them. It should be possible to export an iCal file with the relevant events and host it entirely on our website.

Agreed.  If someone can create a single iCal file that can handle the 1st/3rd logic....

**Comment by JeremyRand** [comment]
My iCal skills are nonexistent; can we just put the meeting times themselves (in text) on the website for now, and worry about calendar client support later?

**Comment by josephbisch** [comment]
Is something like this what you mean by iCal file? It successfully imports into Google Calendar for me. Let me know if you would rather that I upload the file itself somewhere instead of pasting the contents here.

```
BEGIN:VCALENDAR
BEGIN:VEVENT
DTSTART:20150207T170000Z
DTEND:20150207T200000Z
RRULE:FREQ=MONTHLY;INTERVAL=1;BYDAY=1SA,3SA
DESCRIPTION:
LOCATION:#namecoin-dev on freenode
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Namecoin dev meetings on irc in #namecoin-dev on freenode.
TRANSP:TRANSPARENT
END:VEVENT
END:VCALENDAR
```

I allocated three hours for the meeting based on my past experience. You might want to change the summary to make it less verbose, because it shows up as the title of the event in Google Calendar. I don't know exactly what all the fields do, because I put this together from various Google searches, but this appears to work.

**Comment by indolering** [comment]
@josephbisch that looks great to me!

**Comment by JeremyRand** [comment]
Can we upload that file to namecoin.org via GitHub?

**Comment by josephbisch** [comment]
I don't know if you are asking me for permission to upload the file, or if you are asking someone if it is technically possible to do so, but you have my permission.

**Comment by JeremyRand** [comment]
I was mainly asking for someone to upload it.  I figured it was fine with you.  :-)
